### PR TITLE
4187 use permission change command

### PR DESCRIPTION
**Pull request by VakarisZ**
# What does this PR do?

Issue #4187

Add any further explanations here.

## PR Checklist
* [ ] Have you added an explanation of what your changes do and why you'd like to include them?
* [ ] Is the TravisCI build passing?
* [ ] Was the CHANGELOG.md updated to reflect the changes?
* [ ] Was the documentation framework updated to reflect the changes?
* [ ] Have you checked that you haven't introduced any duplicate code?

## Testing Checklist

* [ ] Added relevant unit tests?
* [ ] Do all unit tests pass?
* [ ] Do all end-to-end tests pass?
* [ ] Any other testing performed?
    > Tested by {Running the Monkey locally with relevant config/running Island/...}
* [ ] If applicable, add screenshots or log transcripts of the feature working

### CODE REVIEW

**Review by mssalvatore — COMMENTED** [review]
Does MSSQL also need fixing?

**Comment by VakarisZ** [comment]
mssql only runs on windows

File: monkey/agent_plugins/exploiters/hadoop/src/hadoop_command_builder.py [hunk 1]
```diff
@@ -78,11 +79,17 @@ def _build_linux_hadoop_command(
         download_url=agent_download_url,
     )
 
+    chmod_options = LinuxSetPermissionsOptions(
+        agent_destination_path=agent_destination_path, permissions=0o700
```
Comment (VakarisZ) [inline monkey/agent_plugins/exploiters/hadoop/src/hadoop_command_builder.py:83]: I think it's better to not specify the default

Comment (mssalvatore) [reply monkey/agent_plugins/exploiters/hadoop/src/hadoop_command_builder.py:83]: I was thinking that "Explicit is better than implicit".

**Pull request merged**